flyteidl/protos/flyteidl/admin/agent.proto

```diff
@@ -11,7 +11,6 @@ import "flyteidl/core/metrics.proto";
 import "google/protobuf/duration.proto";
 import "google/protobuf/timestamp.proto";
 
-
 // The state of the execution is used to control its visibility in the UI/CLI.
 enum State {
```

Comment:
why is it still in admin/agent.proto. I am ok, but just seems odd

Reply:
we already deprecated it